src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlCodeGeneration.cs

```diff
@@ -176,6 +177,11 @@ public XamlCodeGeneration(GeneratorExecutionContext context)
 				_xamlResourcesTrimming = xamlResourcesTrimming;
 			}
 
+			if (bool.TryParse(context.GetMSBuildPropertyValue("IsUnoHead"), out var isUnoHead))
+			{
+				_isUnoHead = isUnoHead;
+			}
```

Comment:
I guess it should work this way?

```suggestion
			_ = bool.TryParse(context.GetMSBuildPropertyValue("IsUnoHead"), out _isUnoHead)
```

Reply:
The idea here is to avoid overriding a default value (since TryParse will forcefully set the out value) but in this case it would not change much, but we could refactor this block to use simpler parsing methods for sure, using refs instead of out. I'm not going to do it for this particular PR for time constraints, but we can in another PR.